app/Jobs/FetchNftActivity.php

```diff
@@ -89,6 +100,14 @@ public function handle(): void
             FetchNftActivity::dispatch($this->nft)->onQueue(Queues::SCHEDULED_WALLET_NFTS);
         }
 
+        Log::info('FetchNftActivity Job: Handled', [
+            'address' => $collection->address,
+            'network' => $collection->network->id,
+            'token_number' => $this->nft->token_number,
+            'upserted_activities' => $upserted,
```

Comment:
maybe include a count only, otherwise there will be lots of activities logged which we can view in the db already 🤔